sdcm/provision/common/utils.py

```diff
@@ -152,6 +152,8 @@ def install_syslogng_service():
         SYSLOG_NG_INSTALLED=""
         if yum --help 2>/dev/null 1>&2 ; then
             if rpm -q syslog-ng ; then
+                rm /etc/syslog-ng/syslog-ng.conf  # Make sure we have default syslog-ng.conf
```

Comment:
why do we need this reinstall ? why isn't it enough we have syslog-ng installed ?